inc/classes/sections/class-astra-customizer-home-page-banner-configs.php

```diff
@@ -0,0 +1,235 @@
+<?php
+/**
+ * Astra Theme Customizer Configuration Home Page Banner.
+ *
+ * @package Home Page Banner for Astra Theme
+ * @since x.x.x
+ */
+
+// No direct access, please.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( class_exists( 'Astra_Customizer_Config_Base' ) ) {
```

Comment:
We can do a negative check and return early instead of wrapping the whole class in the if condition.

Reply:
Done.